paymentsheet/src/main/java/com/stripe/android/link/LinkActivityViewModel.kt

```diff
@@ -271,3 +278,16 @@ internal sealed interface ScreenState {
 }
 
 internal class NoArgsException : IllegalArgumentException("NativeLinkArgs not found")
+
+private val NativeLinkArgs.configurationWithUpdatedCustomerInfo: LinkConfiguration
```

Comment:
nit - I think this is more understandable as an extension function of `LinkConfiguration`, so that it reads:

`args.configuration.withUpdatedCustomerInfo(args.LinkAccount)` - might be a bit more verbose on the other hand so leave it as is if it makes sense!